explainer/explainer11.md

```diff
@@ -1,49 +1,52 @@
+- definitions of basic concepts and architectural constraints;
+- an overview of the WoT building blocks and their interplay; and
+- a guide for mapping the abstract architecture onto a variety of concrete deployment scenarios.
 
-- the basic terminology,
-- use cases and derived requirements the W3C WoT Architecture should satisfy,
-- definitions of the basic concepts and architectural constraints,
-- an overview over the WoT building block documents and their interplay, and
-- a guide for mapping the abstract architecture onto a variety of concrete deployment scenarios
-
-The architecture document lays the foundation for the WoT building block specifications:
+The architecture document lays the foundation for the following WoT building block specifications:
 
 - the <a href="https://w3c.github.io/wot-thing-description/">WoT Thing Description</a> (REC Track),
 - the <a href="https://w3c.github.io/wot-profile/">WoT Profile</a>,
 - the <a href="https://w3c.github.io/wot-binding-templates/">WoT Binding Templates</a>,
 - the <a href="https://w3c.github.io/wot-discovery/">WoT Discovery</a>, and
-- the <a href="https://w3c.github.io/wot-security/">WoT Security and Privacy Considerations</a>.
+- the <a href="https://w3c.github.io/wot-security/">WoT Security and Privacy Guidelines</a>.
 
-Of these, the WoT Security and Privacy Considerations provide a general analysis that cross-cuts the other building blocks, each of which also include their own specific security and privacy considerations.
+Of these, the WoT Security and Privacy Guidelines provide a general analysis that cross-cuts the other building blocks, each of which also include their own specific security and privacy considerations.
 
-## Goals
+In addition, the following optional/informative building blocks are defined:
+* <a href="https://w3c.github.io/wot-scripting-api/">WoT Scripting API</a>, and
+* <a href="https://w3c.github.io/wot-binding-templates/">WoT Binding Templates</a>.
 
-* Simplify the integration of different IoT devices and platforms.
-* Complement existing IoT standards and platforms.
-* Build upon the Web architecture, in particular URIs, registered representation formats (media types), and hypermedia.
-* Allow for IP-based IoT protocols other than HTTP (given they fulfill the Uniform Interface constraint of REST).
-* Allow for different integration patterns: at the device level, gateway/edge node, and the cloud.
+## Goals
+The goals of the W3C Web of Things are to:
+* simplify the integration of different IoT devices and platforms;
+* complement existing IoT standards and platforms;
+* build upon the Web architecture, in particular URIs, registered representation formats (media types), and hypermedia;
+* allow for IP-based IoT protocols other than HTTP (given they fulfill the Uniform Interface constraint of REST); and
+* allow for different integration patterns: at the device level, gateway/edge node, and the cloud.
```

Comment:
Please add "digital twins"

Reply:
I added ", including support for digital twins" after "cloud"